Example/README.md

```diff
@@ -1,17 +1,89 @@
-# Square Mobile Payments SDK Example App
+# 🍩 Donut Counter - A Square Mobile Payments SDK Example App
 
+<p align="center">
+<img src="../Images/donut-counter-home.png" width="300"/> <img src="../Images/donut-counter-take-payment.png" width="300"/>
+</p>
 
-### Assumptions and prerequisites
+Donut Counter is a sample application for building with the [Square Mobile Payments SDK](https://developer.squareup.com/docs/mobile-payments-sdk) on iOS. The app demonstrates how to install and initialize the SDK as well as utilizing the API's and user interfaces provided by the SDK to authorize a merchant, pair and manage card readers, and take a payment. To get started, follow the steps below.
```

Comment:
Maybe add something about it uses SwiftUI, or shows how to integrate into a SwiftUI app?

Reply:
Updated 🙏